tests/python/unittest/test_ndarray.py

```diff
@@ -1495,6 +1495,45 @@ def test_dlpack():
             mx.test_utils.assert_almost_equal(a_np, d_np)
             mx.test_utils.assert_almost_equal(a_np, e_np)
 
+@with_seed()
+def test_ndarray_is_inf():
+    random_dimensions = np.random.randint(3, 5)
+    random_shape = [np.random.randint(2, 5) for i in range(random_dimensions)]
+    data = mxnet.test_utils.rand_ndarray(random_shape,'default')
+    data[0][0][0]=np.inf
```

Comment:
use create data with random dimension, but only set number in three dimensions?

Reply:
This was after discussion with @anirudhacharya about how to verify that cases of infinite, negative infinite and NaN are handled.

Reply:
the dimensions for setting of nan and np.inf values can be randomized too.